pkg/storaged/content-views.jsx

```diff
@@ -206,49 +207,42 @@ function create_tabs(client, target, is_partition) {
     }
 
     function unlock() {
-        if (!client.features.clevis)
-            return unlock_with_passphrase();
-        else {
-            return clevis_unlock()
-                    .then(null,
-                          function () {
-                              return unlock_with_passphrase();
-                          });
-        }
+        var crypto = client.blocks_crypto[block.path];
+        if (!crypto)
+            return;
+
+        return get_existing_passphrase(block, true).then(type => {
+            if (type == "stored") {
+                return (crypto.Unlock("", {})
+                        .catch(() => {
```

Comment:
The difference is in the return value. As written, this returns `undefined`, with a single-line expression, it would "accidentally" return the return value of `unlock_with_passphrase()`. I like to make it expicit that the return value of `unlock_with_passphrase() is not actually used. But, yeah, you people keep telling me otherwise, so *shrug*. :-)

Reply:
Done.